fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerBuildGroupedCountIndexTest.java

```diff
@@ -74,9 +74,11 @@ private static Stream<String> sourceIndexNames() {
         return Stream.of(null, "MySimpleRecord$primary_key", NUM_VALUE_2_INDEX.getName(), STR_VALUE_INDEX.getName());
     }
 
-    static Stream<Arguments> sourceIndexes() {
+    @Nonnull
+    static Stream<Arguments> sourceIndexesAndRandomSeeds() {
         return sourceIndexNames()
-                .map(Arguments::of);
+                .flatMap(sourceIndexName -> OnlineIndexerBuildIndexTest.randomSeeds()
```

Comment:
Note: this will effectively cause it to have 5x the number of iterations. I don't think how `iterations` is flushed out is fully concluded, but is that what is desired here? If so, perhaps worth commenting that we want to run for each sourceIndexName with `iterations` random seeds.

Reply:
I think this differs from the usage in `LuceneIndexMaintainerTest`, but I'm not really sure which is better. That one is annoying in that you have to make up random numbers for the seeds when creating specific arguments. I think if this were aligned, it would be something like:
```
Stream.concat(
    sourceIndexNames.map(source -> Arguments.of(source, 234098),
    RandomizedTestUtils.randomArguments(random -> Arguments.of(randomSource(random), random.nextLong()));
```

but I don't know which is better.
Perhaps better to get a few more tests utilizing the framework and then coalesce, and cleanup the ones that don't align.